### PR TITLE
ENH: `Styler.to_latex` conversion from CSS

### PR DESCRIPTION
**Pull request by attack68**
This is an extension PR to #40422 (most of this code is included there), and must be committed after it.
This PR only adds the `_parse_latex_css_conversion` function and associated `convert_css` argument.

Below is the relevant section added to the user guide:

Screen Shot 2021-04-01 at 16 37 12

### CODE REVIEW

**Comment by simonjayhawkins** [comment]
@attack68 there are some mypy errors.

**Comment by attack68** [comment]
> @attack68 there are some mypy errors.

thanks didn't spot, fixed now.

File: pandas/io/formats/style.py [hunk 1]
```diff
@@ -480,6 +481,10 @@ def to_latex(
             Set to ``True`` to structure LaTeX compatible with the {siunitx} package.
         encoding : str, default "utf-8"
             Character encoding setting.
+        convert_css : bool, default False
```
Comment (jreback) [inline pandas/io/formats/style.py:484]: do we need an option for this? why wouldn't we always do this?

**Review by jreback — CHANGES_REQUESTED** [review]
add this on to an existing whatsnew note the issue number

File: pandas/io/formats/style.py (hunk 1 above)
Comment (moi90) [reply pandas/io/formats/style.py:484]: We wouldn't want to convert if different settings for html and latex are wanted or if more control over the latex formatting is needed. But in that case, a new styler instance could easily be used, right?

Comment (jreback) [reply pandas/io/formats/style.py:486]: right this is by-definition on `.to_latex` so i think its clear.

Comment (attack68) [reply pandas/io/formats/style.py:486]: Styler now allows styling to be coded in LaTeX **or** HTML-CSS format. In each separate case, which are completely flexible in each format, you don't want to convert.
But, if you have created an HTML-CSS Styler and it contains the limited attr-values that can now be converted to LaTeX then you should use the convert option. Shouldn't be set to default though.

**Review by jreback — APPROVED** [review]

**Comment by jreback** [comment]
thanks @attack68

**Comment by jreback** [comment]
@meeseeksdev backport 1.3.x

**Pull request merged**